connectors/overview.mdx

```diff
@@ -51,66 +51,11 @@ to configure them in Arroyo.
 
 ## Connection formats
 
-Some connectors support arbitrary data formats, while others have a fixed format defined
-by the connector. For example, Kafka stores data as a sequence of bytes, which can be
-interpreted in many different ways. When creating a connection it's necessary to specify
-the format of the data so that Arroyo can serialize or deserialize it.
+Arroyo supports a number of different data formats for reading and writing data, including
+JSON, Avro, Parquet, and raw strings. The format is configured via the `format` option
```

Comment:
link to format page here?

Reply:
It's linked at the bottom of the section.